### PR TITLE
Use dartino checkout as dartino sdk

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Comment by danrubel** [comment]
LGTM. Thank you!

**Comment by sigurdm** [comment]
I added one more commit. Could you PTAL?

File: lib/dartino/sdk/dartino_sdk.dart [hunk 1]
```diff
@@ -209,7 +209,21 @@ class DartinoRepo extends DartinoSdk {
...
-    String command = fs.join(fs.join(sdkRoot, 'out', 'ReleaseX64'), osBinName);
+    String binDir;
+    if (["dartfmt", "dartanalyzer", "pub"].contains(binName)) {
+      if (isLinux) {
+        binDir = fs.join('third_party', 'dart-sdk', 'linux', 'bin');
+      } else if (isMac) {
+        binDir = fs.join('third_party', 'dart-sdk', 'mac', 'bin');
+      } else {
+        assert(isWindows);
+        binDir = fs.join('third_party', 'dart-sdk', 'mac', 'bin');
```
Comment (danrubel) [inline lib/dartino/sdk/dartino_sdk.dart]: 'mac' ---> 'win' ?

**Comment by danrubel** [comment]
One nit, but still LGTM

**Comment by sigurdm** [comment]
Good catch!

**Comment by devoncarew** [comment]
I'm assuming this PR is now stale? Closing...

**Pull request closed without merging**